packages/node/src/integrations/tracing/knex.ts

```diff
@@ -0,0 +1,39 @@
+import { KnexInstrumentation } from '@opentelemetry/instrumentation-knex';
+import { SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN, defineIntegration, spanToJSON } from '@sentry/core';
+import type { IntegrationFn } from '@sentry/types';
+import { generateInstrumentOnce } from '../../otel/instrument';
+
+const INTEGRATION_NAME = 'Knex';
+
+export const instrumentKnex = generateInstrumentOnce(
+  INTEGRATION_NAME,
+  () => new KnexInstrumentation({ requireParentSpan: true }),
+);
+
+const _knexIntegration = (() => {
+  return {
+    name: INTEGRATION_NAME,
+    setupOnce() {
+      instrumentKnex();
+    },
+
+    setup(client) {
+      client.on('spanStart', span => {
+        const spanJSON = spanToJSON(span);
+        const spanData = spanJSON.data;
+
+        if (spanData && 'knex.version' in spanData) {
```

Comment:
In knex instr, the prop `knex.version` is added to the span [attribute](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/19454560b49b669c8f9c0cca9fcc3308b415f2b1/plugins/node/opentelemetry-instrumentation-knex/src/instrumentation.ts#L137).